app/components/Navbar/Navbar.tsx

```diff
@@ -124,6 +132,17 @@ export default function Navbar() {
               <GitHubIcon />
             </a>
           </li>
+          <li>
+            <a
+              className="btn-ghost btn hidden md:inline-flex"
+              href={discordUrl}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <span className="mr-2">Discord</span>
+              <DiscordIcon />
+            </a>
+          </li>
```

Comment:
Totally fine if you don't take the suggestion above. However, I would recommend removing the `className="mr-2"` as well as the `<span>` around these titles and instead use `gap-x-2` on the `<a>` element itself.

The reason for this change is so that once the elements collapse to 2 lines, you don't get them being slightly off center. Easier to show by example.

**Before**
![image](https://user-images.githubusercontent.com/12396812/205668745-c475a6d3-8504-4ea6-8680-d6843d6920f4.png)

**After**
![image](https://user-images.githubusercontent.com/12396812/205668801-d78fb1cd-a4a9-489f-8508-d359afe1fd12.png)

Note: it is incredibly small how off center it is. Change `mr-2` to something like `mr-6` to see a more pronounced version of the problem.